src/Fixer/ConfigurationDefinitionFixerInterface.php

```diff
@@ -15,6 +15,8 @@
 use PhpCsFixer\OptionsResolver;
 
 /**
+ * Maybe better the other way around, i.e. now there is no upgrade path to the new interface without deprecation warnings
```

Comment:
I would deprecate it here but in 2.x as soon as 3.0 is released (so no deprecating in 3.0)